DOC-2318: Fix `skip_focus` check to the `ToggleToolbarDrawer` command from 6.4.1 release notes. (#3127)
tinymce/tinymce-docs: modules/ROOT/pages/6.1-release-notes.adoc

This update fixes this bug. When the Comments plugin loads, the comment model data is no longer placed into an undo level and no Undo actions will present as available until there are actual changes available to undo.

> ==== Opening the comments sidebar via an editor command did not respect the `skipFocus` argument

Reviewer: ```suggestion
==== Opening the comments sidebar via an editor command did not respect the `grabFocus` argument
```

> If the Comments sidebar was opened with an editor command and `skipFocus` was set to true — `editor.execCommand("ToggleSidebar", false, "showcomments", { skipFocus: true });` — the sidebar still stole focus.

Reviewer: ```suggestion
If the Comments sidebar was opened with an editor command and `grabFocus` was set to true — `editor.execCommand("'tc-open-comment'", false, "showcomments", { grabFocus: true });` — the sidebar still stole focus.
```
For some reason this was documented incorrectly.

> If the Comments sidebar was opened with an editor command and `grabFocus` was set to true — `editor.execCommand("'tc-open-comment'", false, "showcomments", { grabFocus: true });` — the sidebar still stole focus.

Reviewer 2: ```suggestion
If the Comments sidebar was opened with an editor command and `grabFocus` was set to true — `editor.execCommand("tc-open-comment", false, "showcomments", { grabFocus: true });` — the sidebar still stole focus.
```

How about `the sidebar still had the focus` instead of `the sidebar still stole focus`? As my understanding is the editor should have the focus instead of the comments sidebar when `grabFocus` is true

> This update corrects this: `skipFocus: true` is honored, as expected, and the sidebar does not steal focus.

Reviewer: ```suggestion
This update corrects this: `grabFocus: true` is honored, as expected, and the sidebar does not steal focus.
```

> This update corrects this: `grabFocus: true` is honored, as expected, and the sidebar does not steal focus.

Reviewer 2: ```suggestion
This update corrects this: `grabFocus: true` is honored, as expected, and the sidebar does not have the focus.
```
